### ISSUE TITLE
consider further migration strategies

### ISSUE THREAD

**Issue by gernotstarke** [aim42/aim42#160]
from the German wikipedia:
http://de.wikipedia.org/wiki/Migrationsstrategien_%28Informationstechnik%29

**Comment by gernotstarke** [comment]
Big Bang Approach (Cold Turkey Strategy), 
Chicken Little Strategy,
 Database First Approach (Forward Migration Method), 
Database Last Approach (Reverse Migration Method),
 Composite Database Approach (Parallel Database Strategy) und 
Butterfly Methodology

**Comment by kitenco** [comment]
I can do this.

But I don't know how to change the graphic.
I would draw Database First, Database Last and Composite Database Approach as sub items of Chicken Little

And I would remove the connection from Chicken Little to Data Migration,
because Chicken Little is not only a data migration approach

**Comment by feststelltaste** [comment]
There are plenty of additional approaches in the [OORP book](http://scg.unibe.ch/download/oorp/), page 182ff.

![grafik](https://user-images.githubusercontent.com/1470822/48734068-a7af1280-ec44-11e8-866d-f3d4e05489a9.png)

**Comment by gernotstarke** [comment]
thanx - imho most of these represent "practices" (tactical stuff), some are fundamentals ("always have running version, prototype, use profiler before optimizing), some are VERY generic ("bridge to new town").

For me it feels that our approaches are already more mature that the OORP ideas... nevertheless a timeless book which is worth considering for us.

**Comment by feststelltaste** [comment]
Yes, right, after reconsidering it, we really go already deeper than OORP here.
